src/modules/dome.c

```diff
@@ -8,6 +8,20 @@ PROCESS_exit(WrenVM* vm) {
   wrenSetSlotNull(vm, 0);
 }
 
+internal void
+PROCESS_args(WrenVM* vm) {
+  ENGINE* engine = (ENGINE*)wrenGetUserData(vm);
+  
+  wrenEnsureSlots(vm, 2);
+  wrenSetSlotNewList(vm, 0);
+
+  for (int i = 0; i < engine->argc; i++)
```

Comment:
If this code-block was taken from the Wren-CLI project, we should make some note of that fact here, for licensing reasons.